packages/core/src/accessanalyzer/vue/policyChecks.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as fs from 'fs'
+import { getLogger, Logger } from '../../shared/logger'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { VueWebview } from '../../webviews/main'
+import { ExtContext } from '../../shared/extensions'
+//import { telemetry } from '../../shared/telemetry/telemetry'
```

Comment:
Remove this commented out unused import.

Reply:
We will be implementing Telemetry soon, kept it there as a reminder